Directory.Build.targets

```diff
@@ -5,7 +5,7 @@
     <ReadMeExists Condition="Exists('$(ReadMePath)')">true</ReadMeExists>
     <PackageReadmeFile Condition="'$(PackageReadmeFile)' == '' And '$(ReadMeExists)' == 'true'">README.md</PackageReadmeFile>
 
-    <TestArchiveTestsDir Condition="'$(IsWorkloadTestProject)' == 'true'">$(TestArchiveTestsDirForWorkloadTests)</TestArchiveTestsDir>
+    <TestArchiveTestsDir Condition="'$(TestArchiveTestsDir)' == '' and '$(IsWorkloadTestProject)' == 'true'">$(TestArchiveTestsDirForWorkloadTests)</TestArchiveTestsDir>
```

Comment:
This is weird that this property is set in both the `.props` and the `.targets`. Why both?

Can this condition ever be true since it is already defaulted in the `.props`?